.github/actions/deploy/action.yml

```diff
@@ -83,7 +83,7 @@ runs:
         secrets: |
           /secrets/app/secrets.json=skid-secrets:latest
         flags: |
-          --memory=512Mi
+          --memory=1G
```

Comment:
**question**: isn't this 1Gi

Reply:
<img width="442" alt="image" src="https://github.com/user-attachments/assets/21f720f6-299e-418f-b92c-897b7ea898ac">

I'm not sure what the difference, if any, is between `1G` and `1Gi` but it seems to have worked:

<img width="442" alt="image" src="https://github.com/user-attachments/assets/5e066f6e-4884-4033-bc7a-939c76a20aa3">